nginx.conf

```diff
@@ -0,0 +1,21 @@
+events {
+}
+http {
+  # This path may be different on your local computer depending on how you
+  # installed nginx.
+  include /usr/local/etc/nginx/mime.types;
```

Comment:
the other solution is to include a mime.types and relpath it as well

Reply:
You mean committing `mime.types` to the repository? I like that idea: I could see that being easier for contributors so that they don't have to modify paths. But we'd also miss any NGINX updates to `mime.types`, right (or have to monitor for them)? That might not be much of a concern though, as I doubt `mime.types` updates in big ways that often...